src/lsm/lsm_worker.c

```diff
@@ -58,8 +60,7 @@ __lsm_worker_general_op(
 	/*
 	 * Return if this thread cannot process a bloom, drop or flush.
```

Comment:
comment needs updating ? We are also checking for "evict" and "switch" here. We don't process "switch" in the code that follows. What happens if only WT_LSM_WORK_SWITCH is set. Earlier we would have returned WT_NOTFOUND, now we will pop an entry and free it. I don't know the code very well so it might all be okay to do.